include/grpc/grpc_security.h

```diff
@@ -731,6 +731,19 @@ GRPCAPI int grpc_tls_credentials_options_set_cert_request_type(
     grpc_tls_credentials_options* options,
     grpc_ssl_client_certificate_request_type type);
 
+/** Set grpc_tls_server_verification_option field in credentials options
+    with the provided type.options should not be NULL.
```

Comment:
What do you mean by `type.options`?

Reply:
Typo in comment. I meant `with the provided server_verification_option. options should not be NULL` . Similar to grpc_tls_credentials_options_set_cert_request_type. Will fix it.